static/css/add_movie.css

```diff
@@ -1,29 +1,30 @@
 ul.nav.nav-pills {
-    justify-content: space-between;
+  justify-content: space-between;
 }
 
 ul.nav.nav-pills li {
-    flex: 1;
-    text-align: center;
-    margin-bottom: 0;
+  flex: 1;
+  text-align: center;
+  margin-bottom: 0;
 }
 
 @media (max-width: 820px) {
-   ul.nav.nav-pills li {
-        width: calc(25% - 15px);
-        flex: auto;
-        margin-bottom: 10px;
-    }
+  ul.nav.nav-pills li {
+    width: calc(25% - 15px);
+    flex: auto;
+    margin-bottom: 10px;
+  }
 }
 
 @media (max-width: 560px) {
-    ul.nav.nav-pills li {
-        width: calc(50% - 15px);
-    }
+  ul.nav.nav-pills li {
+    width: calc(50% - 15px);
+  }
 }
 
 ul#movies {
   max-width: 100%;
+  max-height: inherit !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=5386949)